client/src/Components/DropMenu/index.js

```diff
@@ -0,0 +1,117 @@
+import React, { Fragment, Component } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {
+	Menu,
+	MenuItem,
+	ListItemIcon,
+	ListItemText,
+	IconButton,
+} from '@material-ui/core';
+import {
+	ArrowDropDown as ArrowDropDownIcon,
+	ExitToApp as ExitToAppIcon,
+	HighlightOff as HighlightOffIcon,
+	AccountBox as AccountBoxIcon,
+	SentimentSatisfied,
+} from '@material-ui/icons';
+import style from './style';
+import Dialog from '../Dialog';
+
+const StyledMenu = withStyles({
+	paper: {
+		border: '1px solid #d3d4d5',
+	},
+})((props) => (
+	<Menu
+		elevation={0}
+		getContentAnchorEl={null}
+		anchorOrigin={{
+			vertical: 'bottom',
+			horizontal: 'center',
+		}}
+		transformOrigin={{
+			vertical: 'top',
+			horizontal: 'center',
+		}}
+		{...props}
+	/>
+));
+
+class DropMenu extends Component {
+	state = {
+		anchorEl: null,
+		open: false,
+	};
+
+	handleClick = (event) => {
+		this.setState({ anchorEl: event.currentTarget });
+	};
+
+	handleClose = () => {
+		console.log('here');
+		this.setState({ anchorEl: null });
+	};
+
+	handleMenuClick = (event, index) => {
+		if (index === 1) {
+			return this.props.history.push('/user/:userId/profile');
+		} else if (index === 3) {
+			return this.props.history.push('/');
+		} else this.setState({ open: true });
+		event.preventDefault();
```

Comment:
Usually preventDefault would be first, but maybe there's a reason it not here.

Fun fact: If you're returning from the precedent conditional block you can skip the `else` and simplify the cascading logic.